pkg/expr/transform.go

```diff
@@ -69,7 +94,20 @@ func WrapTransformData(ctx context.Context, query *tsdb.TsdbQuery) (*tsdb.Respon
 
 // TransformData takes Queries which are either expressions nodes
 // or are datasource requests.
-func TransformData(ctx context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
+func TransformData(ctx context.Context, req *backend.QueryDataRequest) (r *backend.QueryDataResponse, err error) {
+	start := time.Now()
+	defer func() {
+		var respStatus string
+		switch {
+		case err == nil:
+			respStatus = "success"
+		default:
+			respStatus = "failure"
+		}
+		transformQueryCounter.WithLabelValues(respStatus).Inc()
+		transformQueryHistogram.WithLabelValues(respStatus).Observe(time.Since(start).Seconds())
```

Comment:
@bergquist do you think it would be better to store milliseconds instead of seconds?

Reply:
I would think ms